packages/discord.js/src/structures/Emoji.js

```diff
@@ -56,6 +56,15 @@ class Emoji extends Base {
     return this.id && this.client.rest.cdn.emoji(this.id, this.animated ? 'gif' : 'png');
   }
 
+  /**
+   * @type {?string}
+   * @param {ImageExtension} ext the extension of emoji
+   * @returns {string}
+   */
+  getURL(ext) {
+    return this.id && this.client.rest.cdn.emoji(this.id, ext);
+  }
```

Comment:
i think the position of this function needs to be below all of the getter methods
```suggestion
  /**
   * @param {ImageExtension} ext the extension of emoji
   * @returns {?string}
   */
  getURL(ext) {
    return this.id && this.client.rest.cdn.emoji(this.id, ext);
  }
```
also corrected documentation

Reply:
```suggestion
  /**
   * @param {ImageExtension} ext The extension of the emoji
   * @returns {?string}
   */
  getURL(ext) {
    return this.id && this.client.rest.cdn.emoji(this.id, ext);
  }
```